openfl/utilities/attestation/attestation_utils.py

```diff
@@ -0,0 +1,383 @@
+
+from base64 import b64encode
+import json
+import base64
+import os
+import http
+import requests
+import hashlib
+import logging
+
+from openfl.utilities.attestation.signer_utils import ECDSASigner
+
+
+logger = logging.getLogger(__name__)
+
+
+class AttestedIdentity:
+    """Class to represent an attested identity for a participant enclave.
+    This class holds the MRENCLAVE value, participant name, AVS report token,
+    certificate, and private key for mTLS communication with the collaborator enclave.
```

Comment:
I think this sentence is redundant with the code (and may even become outdated over time). I suggest simply removing it.

Reply:
removed